src/Hbar.ts

```diff
@@ -21,12 +20,12 @@ function convertToTinybar(amount: BigNumber.Value, unit: HbarUnit): BigNumber {
 
 export type HbarUnit = keyof typeof tinybarConversions;
 
-/** The possible denominations of hbar in order by magnitude */
+/** The possible denominations of HBAR in order by magnitude */
```

Comment:
HBAR is the spelling used on Hedera's site so I figured that's what should be used in the SDK.